src/app.rs

```diff
@@ -254,12 +269,69 @@ fn parse_string_to_f64(string: String) -> f64 {
     float
 }
 
+fn move_to(
```

Comment:
Add `#[allow(clippy::too_many_arguments)]` on top of `fn move_to(`
Clippy thinks there's too many arguments; it allows only 7 by default, this should calm it.

Reply:
Done.